tools/log_example.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
```

Comment:
In this `tools` directory, can we have a `requirements.txt` file or some way to get dependencies?

Reply:
The modules in `suntools` only require modules from the Python standard library so a requirements file is not needed to use the parsing tools. However, these examples that use the parsing capabilities and then make plots do require matplotlib, but I'm hesitant to include a requirements file here that's not need for `suntools`.

Reply:
In that case, I think it's fine then.